### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by Es7ven** [Shayanmohajer/ECCV-2024-MSE-Teacher#1]
@deponce @Shayanmohajer , hi
criterion = nn.MSELoss() if opt.loss_func.upper == 'MSE' else nn.CrossEntropyLoss()
The code used to select the loss function is wrong, it should be opt.loss_func.upper() == 'MSE', otherwise the loss function would just be CE loss. i wonder if the results in the paper were obtained by running with the code you posted? Thanks.
